rmgpy/rmg/main.py

```diff
@@ -804,6 +807,41 @@ def execute(self, **kwargs):
         
         self.finish()
     
+    def check_model(self):
+        """
+        Run checks on the RMG model
+        """
+        #check that no two species in core or edge are isomorphic
+        for i,spc in enumerate(self.reactionModel.core.species):
+            for j in xrange(i):
+                spc2 = self.reactionModel.core.species[j]
+                if spc.isIsomorphic(spc2):
+                    raise CoreError('Although the model has completed, species {0} is isomorphic to species {1} in the core.  Please open an issue on GitHub with the following output'.format(spc.label,spc2.label))
+                    
+        
+        for i,spc in enumerate(self.reactionModel.edge.species):
+            for j in xrange(i):
+                spc2 = self.reactionModel.edge.species[j]
+                if spc.isIsomorphic(spc2):
+                    logging.warning('species {0} is isomorphic to species {1} in the edge.  Note this does not affect the generated model.  If you would like to report this to help make RMG better please open a GitHub issue'.format(spc.label,spc2.label))
```

Comment:
I think we need to output adjLists to be able to troubleshoot

Reply:
should be fixed now